server/package.json

```diff
@@ -13,7 +13,8 @@
     "db:seed": "knex seed:run",
     "db:migrate": "knex migrate:make swag",
     "test": "nyc --reporter=html mocha test --exit",
-    "lint:js": "eslint ."
+    "lint:check": "eslint .",
```

Comment:
@BonfaceKilz circle-ci uses the ``lint:js`` command to check for any limiting issues, you changing this will make your frontend linting job fail